### PR TITLE
Add 'igiftcards' to the 'Utilities' category

### LINKED ISSUE #1017: Add 'igiftcards' to the 'Utilities' category
Requesting to add 'igiftcards' to the 'Utilities' category. 

Details follow: 
```yml 
- name: igiftcards
      url: https://igiftcards.nl
      img: 
      region: eu
      country: NL
      bch: Yes
      btc: Yes
      othercrypto: Yes
      doc: Note: several other country domains: fr/de/at/it/es exist as well
      exceptions: 
        text: "Choose Bitcoin (Not cryptocurrencies)"
```

Resources for adding this merchant:
[Link to igiftcards](https://igiftcards.nl)
Maybe you might want to check their Alexa Rank: [https://www.alexa.com/siteinfo/igiftcards.nl](https://www.alexa.com/siteinfo/igiftcards.nl)

- Verify site is legitimate and safe to list.
- Correct data in form if any is inaccurate.

If everything looks okay, Add it to the site:
- Assign to yourself when you begin work.
- Download and resize the image and put it into the proper img folder.
- Add listing alphabetically to proper .yml file.
- Commit changes mentioning this issue number with `closes #[ISSUE NUMBER HERE]`.

### PR DESCRIPTION
**Pull request by accept-bitcoin-cash**
This closes #1017.

### CODE REVIEW

File: _data/utilities.yml [hunk 1]
```diff
@@ -469,3 +469,14 @@ websites:
...
   othercrypto: false
+- name: igiftcards
+  url: https://igiftcards.nl
+  img: igiftcards.png
+  bch: Yes
+  btc: Yes
+  othercrypto: Yes
+  region: eu
+  country: NL
+  doc: Note: several other country domains: fr/de/at/it/es exist as well
```
Comment (EchterAgo) [inline _data/utilities.yml]: Needs quotes here I think

Comment (kenman345) [reply _data/utilities.yml]: I shall work on the automation to identify when needed and add the quotes.

If that seems to be more complicated than one would hope, I can just add the quotes every time, wouldn’t be any harm

**Comment by kenman345** [comment]
Lol, needs and image, probably should see about adding a label for that when I open a Pr when it’s missing, in case other issues blow up the listing as well

**Comment by accept-bitcoin-cash** [comment]
Added igiftcards.png to utilities.

**Pull request merged**